pkg/k8shandler/migrations/migrations.go

```diff
@@ -33,20 +32,27 @@ func (mr *migrationRequest) RunKibanaMigrations() error {
 
 	indices, err := mr.esClient.GetAllIndices(kibanaIndex)
 	if err != nil {
-		return fmt.Errorf("failed to get `.kibana` index health before running migrations: %s", err)
+		return kverrors.Wrap(err, "failed to get index health before running migrations",
+			"index", kibanaIndex)
```

Comment:
The error text was wrong in the first place. It should be `failed to get indices before running migrations`, `alias`, kibanaIndex